src/large_pages/node_large_page.cc

```diff
@@ -258,23 +258,20 @@ struct text_region FindNodeTextRegion() {
 
 #if defined(__linux__)
 bool IsTransparentHugePagesEnabled() {
-  std::ifstream ifs;
-
   // File format reference:
   // https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/mm/huge_memory.c?id=13391c60da3308ed9980de0168f74cce6c62ac1d#n163
-  ifs.open("/sys/kernel/mm/transparent_hugepage/enabled");
-  if (!ifs) {
+  std::ifstream ifs("/sys/kernel/mm/transparent_hugepage/enabled", std::ios::in);
```

Comment:
```suggestion
  std::ifstream ifs("/sys/kernel/mm/transparent_hugepage/enabled",
                    std::ios::in);
```
to fix the lint issue because 80 is the max char limit for a line

Reply:
Thanks!  I decided to extract the path filename into a separate variable, which also allows the code to fit within the width limit.  There was a secondary consideration too: declaring the type of the `filename` variable might help folks reading the `ifstream`-related documentation to determine which constructor overload's being called.